projects/nae-example-app/src/app/doc/ldap-group-roles-assign/ldap-group-roles-assign.component.scss

```diff
@@ -0,0 +1,25 @@
+.full-height {
```

Comment:
why does this component specify these classes, if they are unused?

Reply:
Done